### DISCUSSION TITLE
Dịch Tesseract

### DISCUSSION THREAD

**Discussion by quangnhat185** [mlbvn/handson-ml2-vn#62] [Terminology] [answered]
Trong chương 8 có thuật ngữ `Tesseract` là dạng 4 chiều của khối lập phương. Mọi người cho ý kiến về cách dịch từ này nhé? Mình có đề nghị dịch từ này là `khối lập phương 4 chiều` nhưng thấy hơi dài. 

cc @mlbvn/handson-ml2

**Comment by lkhphuc** [comment, marked as answer]
Mình đề xuất nên giữ nguyên là Tesseract như trên wiki nếu tiếng việt chưa có từ thay thế.

Từ `cube` trong hình học là `khối vuông`, trong đại số là `lập phương`, giống từ `square` vừa là `hình vuông` vừa là `bình phương`.
Nên từ cube trong `4-dimensional cube` mình thấy hiểu nghĩa hình học là `khối` thôi chứ không nên thêm `lập phương` vào thành `khối lập phương 4 chiều` vừa dài mà lại không chuẩn lắm.

**Comment by rootonchair** [comment]
Em đồng tình với hướng giữ nguyên Tesseract
